README.md

```diff
@@ -76,13 +76,54 @@ There are several other aspects you can optionally configure.  Here is the full
           <skip>false</skip>
           <format>JSON</format>
           <filename>openapi-2.0</filename>
+          <oauth2 />
         </configuration>
       </execution>
     </executions>
   </plugin>
 ```
 
-# Still in development
+### OAuth2 Configuration
+
+By default the plugin will not generate any security information.  You can optionally provide
+it using the `<oauth2 />` element.  Here is an example:
+
+```xml
+  <plugin>
+    <groupId>com.github.berkleytechnologyservices.restdocs-spec</groupId>
+    <artifactId>restdocs-spec-maven-plugin</artifactId>
+    <version>${restdocs-spec.version}</version>
+    <executions>
+      <execution>
+        <goals>
+          <goal>generate</goal>
+        </goals>
+        <configuration>
+          <oauth>
```

Comment:
@klieber does this want to be `<oauth2>`?

Reply:
Yep, guess I should have waited for your review.